nipype/interfaces/fsl/dti.py

```diff
@@ -222,7 +222,7 @@ def _list_outputs(self):
         for k in multi_out:
             outputs[k] = []
 
-        for i in xrange(self.inputs.n_fibres + 1):
+       for i in xrange(1, self.inputs.n_fibres + 1):
```

Comment:
@mattfeld - could you add the extra space in front of the `for` for indentation.

also could you please update the CHANGES file?